### PR TITLE
Edit auto-detect Bitcoin datadir path

### LINKED ISSUE #3: Improve rpc credentials support
- Search in default folders for Linux, Mac and Windows
- Parse .cookie file if it is present
- Parse either rpcuser & rpcpassword or one-line credentials from `bitcoin.conf`
- Add an option to provide a path to bitcoin folder in the config menu

### PR DESCRIPTION
**Pull request by ben-kaufman**
Close #3 
Allow the user to specify his Bitcoin Core data directory path manually.

After though - maybe "Auto-detect" is not the best phrasing for the feature, as what it actually does is load from folder (which if is the default one, will result in automatic detection). Might make sense to change it to something like "Load from folder"? But not sure if that's not a bit confusing as well...

### CODE REVIEW

**Comment by stepansnigirev** [comment]
Looks good.
One thing I noticed: if I provide a path to the folder using user's home folder (`~/.bitcoin` for example) it doesn't work.
It can be resolved with `os.path.expanduser(path)`

**Comment by stepansnigirev** [comment]
Also delete wallet now throws an error:
```
Exception: 'str' object has no attribute 'alias'

Details:
Traceback (most recent call last):
  File "/Users/ss/dev/web/forks/ben_kaufman/.venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/ss/dev/web/forks/ben_kaufman/.venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/ss/dev/web/forks/ben_kaufman/.venv/lib/python3.8/site-packages/flask_login/utils.py", line 269, in decorated_view
    return func(*args, **kwargs)
  File "/Users/ss/dev/web/forks/ben_kaufman/src/cryptoadvance/specter/controller.py", line 1033, in wallet_settings
    app.specter.wallet_manager.delete_wallet(
  File "/Users/ss/dev/web/forks/ben_kaufman/src/cryptoadvance/specter/wallet_manager.py", line 179, in delete_wallet
    logger.info("Deleting {}".format(wallet.alias))
AttributeError: 'str' object has no attribute 'alias'
```

File: src/cryptoadvance/specter/controller.py [hunk 1]
```diff
@@ -1023,7 +1030,9 @@ def wallet_settings(wallet_alias):
             wallet.keypoolrefill(wallet.change_keypool, wallet.change_keypool + delta, change=True)
             wallet.getdata()
         elif action == "deletewallet":
-            app.specter.wallet_manager.delete_wallet(wallet)
+            app.specter.wallet_manager.delete_wallet(
+                app.specter.bitcoin_datadir, wallet
+            )
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/controller.py:1035]: Agruments: other way around: `(wallet, app.specter.bitcoin_datadir)`

Comment (ben-kaufman) [reply src/cryptoadvance/specter/controller.py]: Opps, fixed.

**Pull request merged**